package.json

```diff
@@ -292,8 +293,7 @@
 		"@automattic/newspack-blocks/swiper@4.5.1": "patch:swiper@4.5.1#.patches/swiper@4.5.1.diff",
 		"@automattic/newspack-blocks/prettier": "npm:wp-prettier@2.6.2-beta-1",
 		"@types/react": "^17.0.39",
-		"keytar@npm:7.7.0/node-addon-api": "3.1.0",
-		"lzma-native": "^8.0.5"
```

Comment:
I think we need keep "lzma-native": "^8.0.5" here, otherwise `yarn install` is starting to fail for me.
` YN0009: │ lzma-native@npm:6.0.1 couldn't be built successfully ` - it's trying to build `6.0.1` which is unable to download (404).

Reply:
Interesting! I have an M1 chip and it wasn't causing issues.